src/Symfony/Component/Mailer/SentMessage.php

```diff
@@ -29,6 +29,8 @@ class SentMessage
      */
     public function __construct(RawMessage $message, SmtpEnvelope $envelope)
     {
+        $message->checkValid();
```

Comment:
wouldn't `checkValidity()` be a bit easier to grasp?
or `ensureValid()`?

Reply:
`validate()` maybe?

Reply:
I've changed it to `ensureValidity()`